openlibrary/templates/books/edit.html

```diff
@@ -58,7 +58,7 @@ <h2 class="editFormTitle">$:title</h2>
         <h1 class="editFormBookTitle">$this_title</h1>
         $if authors:
             <h3 class="editFormBookAuthors">
-                $:macros.BookByline([(author.name, author.url(), author.birth_date, author.death_date) for author in authors])
+                $:macros.BookByline([{'name': a.name, 'url': a.url()} for author in authors])
```

Comment:
```suggestion
                $:macros.BookByline([{'name': a.name, 'url': a.url()} for a in authors])
```
This is causing errors in book edit pages.  Did you test this?  Also, the birth and death dates would be useful to librarians while they are editing.

Reply:
Managing my changes in git was a bit of a mess, so I must've gotten this mixed up somewhere.
Will fix, add the birth/date date here, and test.